core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/managers/StatisticsManager.scala

```diff
@@ -1,21 +1,23 @@
 package edu.uci.ics.amber.engine.architecture.worker.managers
 
-import edu.uci.ics.amber.engine.architecture.worker.statistics.{WorkerState, WorkerStatistics}
+import edu.uci.ics.amber.engine.architecture.worker.statistics.WorkerStatistics
 import edu.uci.ics.amber.engine.common.SinkOperatorExecutor
 import edu.uci.ics.texera.workflow.common.operators.OperatorExecutor
 
+import scala.collection.mutable
+
 class StatisticsManager {
   // DataProcessor
-  private var inputTupleCount: Long = 0
-  private var outputTupleCount: Long = 0
+  private var inputTupleCount: mutable.Map[Int, Long] = mutable.Map()
+  private var outputTupleCount: mutable.Map[Int, Long] = mutable.Map()
```

Comment:
please use `PortIdentity` instead of `Int`.

Reply:
Fixed.